_posts/scyrill/2021-11-26-scyrill-meetup-writeup.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+author: scyrill
+title: "Stephanie's Class Hack"
```

Comment:
This won't hurt anything code-wise, but it might be confusing to have this post named for a different assignment. Maybe change it to something like "Stephanie's Meetup Writeup"?

Reply:
Thanks Misha, I think this was a mistake because I pasted it from another assignment because I didn't want to get the format wrong. I will change this right away! @MishaCienfuegos

Reply:
That makes perfect sense, so easy to do! Thanks for making the switch. :)